.travis.yml

```diff
@@ -8,6 +8,7 @@ matrix:
       sudo: true
 
 install:
```

Comment:
Good implementation, I see you have outsourced the secret key to .env

Reply:
Good use of  dj_database_url.config method to return the connection dictionary populated with all the data specified in the URL.